compiler/semcall.nim

```diff
@@ -367,31 +358,48 @@ proc semResolvedCall(c: PContext, n: PNode, x: TCandidate): PNode =
 
 proc canDeref(n: PNode): bool {.inline.} =
   result = n.len >= 2 and (let t = n[1].typ;
-    t != nil and t.skipTypes({tyGenericInst, tyAlias}).kind in {tyPtr, tyRef})
```

Comment:
Wrong diff again here. These lines display as "outdated", so perhaps there is some explanation for this behavior.